rspace/src/test/scala/coop/rchain/rspace/StorageExamplesTests.scala

```diff
@@ -281,6 +281,19 @@ class InMemoryStoreStorageExamplesTests extends StorageExamplesTests {
   }
 }
 
+class ImmutableInMemoryStoreStorageExamplesTests extends StorageExamplesTests {
+
+  override def withTestStore(f: T => Unit): Unit = {
```

Comment:
All the implementations look pretty much the same, the only difference being the creation of the store. Maybe it would make sense to move common code somewhere?